SUMMARY.md

```diff
@@ -144,8 +144,6 @@
 * [How do I copy a map to another project](faq/how-do-i-copy-a-map-to-another-project.md)
 * [How do I control label size while scaling element size?](faq/how-do-i-control-label-size-while-scaling-element-size.md)
 * [How do I control text wrapping?](faq/how-do-i-control-text-wrapping.md)
-* [How do I convert a project from private to public?](faq/how-do-i-convert-a-project-from-private-to-public.md)
```

Comment:
I think we should keep the individual FAQs even if the information is consolidated in other guides. By removing them from the index we just make it harder for people to find the answers they're looking for.

Reply:
Good call, I'll restore sometime today